pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: andax
 description: Interactive branching narratives.
 publish_to: "none"
-version: 0.1.0+8
+version: 0.2.0+9
```

Comment:
I am just curious, based on what do you assign the build number? Funny that Dart itself uses it, but it is not mentioned in https://dart.dev/tools/pub/versioning at all

Reply:
The build number is just incremented whenever there's *any* version increment.

Reply:
Yeah, but I don't think this is how it works. The suffix indicates a [prerelease version](https://dart.dev/tools/pub/publishing#publishing-prereleases), which indicates instability.